crates/forge/src/lib.rs

```diff
@@ -236,7 +236,7 @@ pub async fn run(
     let test_crates = test_crates
         .into_iter()
         .map(|tc| tests_filter.filter_tests(tc))
-        .collect_vec();
+        .collect::<Result<Vec<CompiledTestCrate<RawForkConfig>>>>()?;
```

Comment:
We can use our alias
```suggestion
        .collect::<Result<Vec<CompiledTestCrateRaw>>>()?;
```